tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
+"""
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority, publisher_id, msg):
```

Comment:
I have made required changes in the log_utils file, it should be now able to handle dynamic payload format, please do review it and let me know if any changes are required